system.json

```diff
@@ -1,7 +1,7 @@
 {
-    "id": "hero6efoundryvttv2",
-    "name": "hero6e-foundryvtt-v2",
-    "title": "Hero System 6e (Unofficial) v2",
+    "id": "hero6e-foundry-vtt",
+    "name": "hero6e-foundry-vtt",
+    "title": "Hero System 6e (Repo)",
```

Comment:
Not sure we want these changes in to main branch

Reply:
No indeed. I'll put those back to normal (that was testing purposes)
However it reminds me of a question I had: should the id be changed to fit the folder or should the folder be changed to match the id?

Reply:
fixed in https://github.com/dmdorman/hero6e-foundryvtt/pull/938/commits/a2d1ac7d6dca5d5173f10bc5ff97e7c8a6345dae